context/app/static/js/components/home/HuBMAPDatasetsChart/queries.ts

```diff
@@ -0,0 +1,168 @@
+import { includeOnlyDatasetsClause } from 'js/helpers/queries';
+
+import type { SearchRequest } from '@elastic/elasticsearch/lib/api/types';
+import { getOrganTypesCompositeAggsQuery } from 'js/shared-styles/charts/HorizontalStackedBarChart/utils';
+
+// Query for looking up the list of organs for the Y axis
+const organTypesQuery = {
+  size: 0,
+  query: includeOnlyDatasetsClause,
+  aggs: {
+    organ_types: { terms: { field: 'origin_samples.mapped_organ.keyword', size: 10000 } },
+  },
+};
+
+interface OrganTypesQueryAggs {
+  organ_types: {
+    buckets: {
+      doc_count: number;
+      key: string;
+    }[];
+  };
+}
+
+// Query for looking up the mapping of dataset types to assay display names
+const datasetTypeMapQuery: SearchRequest = {
+  size: 0,
+  query: includeOnlyDatasetsClause,
```

Comment:
```
  size: 0,
  query: includeOnlyDatasetsClause,
```
seems to be used in most of these search request queries. Could it be pulled out?

Reply:
I can make that pair of attributes an object and spread it where needed, similar to the `overflowCss` in panel styles 👍🏻